lib/plaid_handler.ex

```diff
@@ -1,34 +1,39 @@
-defmodule Plaid.Utils do
+defmodule PlaidHandler do
```

Comment:
```suggestion
defmodule Plaid.Handler do
```

Otherwise you are taking an extra namespace from `Plaid` itself

Reply:
Good change and quite right.